modules/apps/object/object-admin-rest-test/src/testIntegration/java/com/liferay/object/admin/rest/resource/v1_0/test/ObjectDefinitionResourceTest.java

```diff
@@ -1775,6 +1778,50 @@ private ObjectDefinition _randomModifiableSystemObjectDefinition()
 		return objectDefinition;
 	}
 
+	private void _testGetObjectDefinitionSettingsExternalReferenceCodes()
```

Comment:
@victorKammerer rename to `_testGetObjectDefinitionWithRootObjectDefinitionExternalReferenceCodes`, this pattern is used in other classes, basically it is:

**method name** + **With** + **variation short description**